# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../metamail/35_mm-2.7-arghhh.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mm2.7/src/bin/showexternal~	Mon Jun 22 10:55:36 1998
+++ mm2.7/src/bin/showexternal	Tue Jun 23 21:16:04 1998
@@ -215,7 +215,7 @@
 	    exit -1
 	endif
 	echo Subject: Automated Mail Server Request > "$NEWNAME"
-	echo To: ${server} >> "$NEWNAME"
+	echo To: "$server" >> "$NEWNAME"
 	echo "" >> "$NEWNAME"
 	sed -e 1,/^\$/d < $bodyfile >> "$NEWNAME"
 	echo "" >> "$NEWNAME"
